interface/modules/custom_modules/oe-module-prior-authorizations/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "juggernautsei/oe-module-prior-authorizations",
+    "description": "Prior Authorizations Module",
+    "type": "openemr-module",
+    "license": "All rights reserved",
+    "authors": [
+        {
+            "name": "Sherwin",
+            "email": "sherwingaddis@gmail.com",
+            "role": "Developer"
+        }
+    ],
+    "keywords": ["openemr", "openemr-module"],
+    "minimum-stability": "stable",
+    "autoload": {
+        "psr-4": {"Juggernaut\\OpenEMR\\Modules\\PriorAuthModule\\": "src/"}
+    },
+    "require": {
+        "openemr/oe-module-installer-plugin": "^0.1.0",
+        "php": ">=7.1",
+        "symfony/event-dispatcher": "^4.4.0",
+        "nyholm/psr7": "^1.4"
```

Comment:
both the symfony/event-dispatcher and nyholm/psr7 packages are already brought in by core composer and there are very notable differences in the versions. Seems like there would be critical issues arising from this.
@adunsulag , any reason why the skeleton module needs these 2 packages that are already in core?

Reply:
I have not run a composer update in a while. When I ran composer install, This is what composer did. It works fine in all of our installs. Composer install created the composer.json. I did not create it.

Reply:
I think the problem is, though, that these old versions of event_dispatcher and psr7 will override newer versions in core when you include the derived autoload.php script.